end2end/tests/reportBuilder.spec.ts

```diff
@@ -2,7 +2,7 @@ import { test, expect } from '@playwright/test';
 
 // When the underlying issue is fixed, we should expect this test to pass.
 // Tests should be tagged with an associated ticket or PR reference
-test.fail('column order is maintained after modifying the search filter', {tag: '@issue:LEAF-4482'}, async ({ page }, testInfo) => {
+test('column order is maintained after modifying the search filter', {tag: '@issue:LEAF-4482'}, async ({ page }, testInfo) => {
```

Comment:
```suggestion
test('column order is maintained after modifying the search filter', async ({ page }, testInfo) => {
```

No need to reference the issue once resolved.